### ISSUE TITLE
Show `serial.log` (and cloud-init.log maybe) during booting the VM

### ISSUE THREAD

**Issue by AkihiroSuda** [lima-vm/lima#960]
### Description

Might be helpful for diagnosis.

Probably we can reuse https://github.com/abiosoft/colima/tree/main/util/terminal .

**Comment by afbjorklund** [comment]
Something like this, perhaps:

https://github.com/charmbracelet/bubbles#viewport

If giving up on survey, that is.

https://github.com/AlecAivazis/survey (archived Sep 24)

**Comment by vsoch** [comment]
Heyo! Is this something that is possible? I'm testing a VM with a few scripts and I really want to see what is going on!

**Comment by afbjorklund** [comment]
You can view the log file and use the socket, this issue was more if it could be shown in the regular CLI interface

**Comment by vsoch** [comment]
>  this issue was more if it could be shown in the regular CLI interface

Yes, understood! I think this is what I was looking for so I can more easily watch.

**Comment by afbjorklund** [comment]
I think it would be nice with _less_ logs during the standard `start` run, not more...

It's very useful to have the logs available afterwards, of course. And to see progress.

**Comment by ChrisPbb** [comment]
Here's a workaround I used to debug a rather annoying typo in a `provision` script.  Simply add this line to the top of each `script` and you'll have a `provision.log` on the host machine exactly where you'd expect (i.e. `~/.lima/INSTANCE`)

```sh
exec >> $LIMA_CIDATA_HOSTHOME_MOUNTPOINT/.lima/$LIMA_CIDATA_NAME/provision.log 2>&1
```
